components/automate-ui/README.md

```diff
@@ -83,13 +83,16 @@ That is a guess on my part, could not really confirm what version one should be
 
```

Comment:
Note: The best way to get through all these files quickly is to go through the commits one by one.